docs/rules/aws_iam_policy_attachment_exclusive_attachment.md

```diff
@@ -0,0 +1,35 @@
+# aws_iam_policy_attachment_exclusive_attachment
+
+Consider alternative resources to `aws_iam_policy_attachment`.
```

Comment:
Get a little bit of the why up here. The section below can expand but this intro should give the reader a 1-2 sentence summary of what _and_ why.